.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "lib/p"]
 	path = lib/p
-	url = git@github.com:ambiata/p
+	url = https://github.com/ambiata/p
 [submodule "lib/disorder"]
 	path = lib/disorder
-	url = git@github.com:ambiata/disorder.hs
+	url = https://github.com/ambiata/disorder.hs
```

Comment:
Yeah, if someone was trying to build a mafia project, they shouldn't necessarily require github ssh credentials to be set up.